extensions/blocks/map/edit.js

```diff
@@ -185,6 +270,37 @@ class MapEdit extends Component {
 						] }
 					/>
 					<PanelBody title={ __( 'Map Settings', 'jetpack' ) }>
+						<BaseControl
+							label={ __( 'Height in pixels' ) }
+							id={ `block-jetpack-map-height-input-${ instanceId }` }
+						>
+							<input
+								type="number"
+								id={ `block-jetpack-map-height-input-${ instanceId }` }
+								onChange={ this.onHeightChange }
+								value={ mapHeight || '' }
+								min={ MIN_HEIGHT }
+								step="10"
+							/>
+						</BaseControl>
+						<RangeControl
+							label={ __( 'Zoom level', 'jetpack' ) }
```

Comment:
Maybe we could rely on `withInputField` and hide the number field when zoom is disabled?

![image](https://user-images.githubusercontent.com/426388/74763883-e507b300-5280-11ea-9770-89819b9f158f.png)

Reply:
I'm not sure what `withInputField` is, `RangeControl` [doesn't have an option](https://github.com/WordPress/gutenberg/blob/5c926f8f67e76c26eed5415ff0360ed06d5c1764/packages/components/src/range-control/index.js#L93-L103) for hiding the number field, that I can see.

Reply:
My bad, I should have checked the source. I was going off the documentation here:
https://developer.wordpress.org/block-editor/components/range-control/#withinputfield

But it seems it was only added a few days ago, so that's not something we'll be able to use right away:
https://github.com/WordPress/gutenberg/pull/19916

Reply:
Huh, cool. I'm inclined to not use it yet, since relatively few sites will have access to it, so it'll just cause JS console warnings about an unknown prop being passed on to the DOM element.